### PR TITLE
Cleanup shell scripts and Dockerfile

### PR DESCRIPTION
**Pull request by dfsnow**
This PR includes some long-overdue linting and formatting on the shell scripts and Dockerfile that makeup this service. It is in preparation for #1. The changes are mostly from shellcheck, shfmt, and hadolint, with some small edits to fix some possibly dangerous behavior.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -23,104 +23,103 @@ LOG_GROUP_NAME="/ccao/jobs/sqoop"
...
     fi
     COUNTER=$((COUNTER + 1))
 done
 echo "Logs successfully uploaded to CloudWatch"
 
 # Remove uploaded log files
-mv ${TEMP_LOG_FILE} ${BACKUP_LOG_FILE}
-rm ${TEMP_LOG_FILE}*
+mv ./"$TEMP_LOG_FILE" ./"$BACKUP_LOG_FILE"
+if [[ -n "$TEMP_LOG_FILE" ]]; then rm ./"$TEMP_LOG_FILE"*; fi
```
Comment (dfsnow) [inline run.sh:125]: Changing this mega dum-dum line so that `rm *` and variants cannot run.

Comment (jeancochrane) [reply run.sh:125]: **[Question, non-blocking]** I don't totally get what this line is for, in what case would `-n "$TEMP_LOG_FILE` evaluate as false? Isn't `TEMP_LOG_FILE` always set to a non-empty string on line 5? I think I'm probably just misunderstanding this!

**Review by jeancochrane — APPROVED** [review]
Nice, thanks for this cleanup!

File: run.sh (hunk 1 above)
Comment (dfsnow) [reply run.sh:125]: Basically only for the case in which someone is playing around with the script and deletes that line, then re-runs the script. Saving us from our future selves.

**Pull request merged**